www/script/project/rtBuilders.js

```diff
@@ -16,7 +16,7 @@ define(function (require) {
         latestRevDict = {},
         tags = new MiniSet(),
         branch_tags = new MiniSet(),// All of the tags that only contain a branch i.e 4.6, Trunk
-        tagAsBranchRegex = /^([0-9].[0-9]|trunk)$/i, // Regex for finding tags that are named the same as branches
+        tagAsBranchRegex = /^(20[0-9][0-9].[0-9]|[0-9].[0-9]|trunk)$/i, // Regex for finding tags that are named the same as branches
```

Comment:
```
Alex Lian

sadly yes, as platforms can come in post-release. Ideally not, realistically yes.
 
I’d suggest a \d+ for the releases. In some future world, we could go monthly or weekly a la Chrome.
```

I think that minor versions might impact the build process (katana configs) but we probably won't need different tags for it. >10 release might happen but overall I think that this approach is safe enough, and we can make a super bullet-proof regex when we need to.

I don't want to be parsing more stuff than we can handle right now :)